admin/class-manage-fonts.php

```diff
@@ -477,6 +523,15 @@ function admin_notice_embed_font_file_error () {
 		<?php
 	}
 
+    function admin_notice_font_asset_removal_error () {
+		$theme_name = wp_get_theme()->get( 'Name' );
+		?>
+			<div class="notice notice-error is-dismissible">
+				<p><?php printf( esc_html__( 'Error removing font asset. WordPress lack permissions to remove these font assets.', 'create-block-theme' ), esc_html( $theme_name ) ); ?></p>
```

Comment:
```suggestion
				<p><?php printf( esc_html__( 'Error removing font asset. WordPress lacks permissions to remove these font assets.', 'create-block-theme' ), esc_html( $theme_name ) ); ?></p>
```

Reply:
Would it always be a permissions issue? Should we make the error statement less specific?

Reply:
This is an error message specific to this problem. There are other messages for other problems. Maybe we can do the error message handling more generic to reduce the number of them but I think we can handle that in another PR.